metricbeat/beater/event.go

```diff
@@ -48,21 +48,32 @@ func (b eventBuilder) build() (common.MapStr, error) {
 		}
 	}
 
-	event = common.MapStr{
-		"@timestamp": timestamp,
-		"type":       typeName,
+	// Checks if additional meta information is provided by the MetricSet under the key _module
+	// This is based on the convention that each MetricSet can provide module data under the key _module
+	moduleData, moudleDataExists := event["_module"]
```

Comment:
`MODULE_DATA` isn't a conventional Go name. Even constants should be [mixed case](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps).